tools/autograd/derivatives.yaml

```diff
@@ -1634,6 +1636,7 @@
   condition: non_differentiable
   self: where(condition, grad, zeros_like(grad))
   other: where(condition, zeros_like(grad), grad)
+  result: where(condition, self_t, zeros_like(other_p)) + where(condition, zeros_like(self_p), other_t)
```

Comment:
Why not `where(condition, self_t, other_t)`?

Reply:
!